python/servo/build_commands.py

```diff
@@ -257,14 +261,15 @@ def build(self, target=None, release=False, dev=False, jobs=None,
             shutil.copy(path.join(self.android_support_dir(), "openssl.makefile"), openssl_dir)
             shutil.copy(path.join(self.android_support_dir(), "openssl.sh"), openssl_dir)
             env["ANDROID_NDK_ROOT"] = env["ANDROID_NDK"]
+            env["RUST_TARGET"] = target
             with cd(openssl_dir):
                 status = call(
                     make_cmd + ["-f", "openssl.makefile"],
                     env=env,
                     verbose=verbose)
                 if status:
                     return status
-            openssl_dir = path.join(openssl_dir, "openssl-1.0.1t")
+            openssl_dir = path.join(openssl_dir, "openssl-1.1.0f")
```

Comment:
We can't suddenly update openssl in the middle of nowhere of a patch that is intended to support arm64, that change is out of scope.